pages/apim/3.x/api-reference/rest-apis-documentation.adoc

```diff
@@ -16,6 +16,10 @@ APIM API exposes a complete set of endpoints for developers wanting to script so
 
 You can access the online endpoint reference or the OpenAPI specification by version:
 
+* 3.10 -- link:/apim/3.x/management-api/3.10/[online reference] - link:/apim/3.x/management-api/3.10/swagger.json[OpenAPI spec]
```

Comment:
Are you sure?

Reply:
good catch, it is from a cherry pick on develop, I remove that line